mongo/database.go

```diff
@@ -778,6 +874,24 @@ func (db *Database) createCollectionOperation(name string, opts ...*options.Crea
 	return op, nil
 }
 
+// mergeCreateViewOptions combines the given CreateViewOptions instances into a single CreateViewOptions in a
+// last-one-wins fashion.
```

Comment:
Optional: see similar for explanation

```suggestion
// last-property-wins fashion.
```